joint_configurations:
  ros__parameters:
    number_of_joints: 7
    names: ["home", "ready", "multi"] # We need to declare them here so we can search them by name
    home:
        positions: [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
        times: [3.0]
    ready:
        positions: [0.1, -0.1,  0.2, -0.5, -0.1,  0.2,  0.0]
        times: [3.0]
    multi:
        positions: [0.1, -0.1,  0.2, -0.5, -0.1,  0.2,  0.0,
                    0.0,  0.0,  0.0,  0.0,  0.0,  0.0,  0.0]
        times: [3.0, 6.0]
